extensions/wikia/PhalanxII/classes/PhalanxPager.class.php

```diff
@@ -75,83 +80,82 @@ function getIndexField() {
 	}
 
 	function getStartBody() {
-		return Html::openElement( 'ul', array( "id" => 'phalanx-block-' . $this->pInx . '-' . $this->id . '-stats' ) );
+		return Html::openElement( 'ul', [ "id" => 'phalanx-block-' . $this->pInx . '-' . $this->id . '-stats' ] );
 	}
 
 	function getEndBody() {
 		return Html::closeElement( 'ul' );
 	}
 
 	function getEmptyBody() {
-		return Html::element('div', array('class' => 'error'), wfMsg('phalanx-no-results'));
+		return Html::element( 'div', [ 'class' => 'error' ], $this->msg( 'phalanx-no-results' )->text() );
 	}
 
 	function formatRow( $row ) {
 		// hide e-mail filters
-		if ( ( $row->p_type & Phalanx::TYPE_EMAIL ) && !$this->app->wg->User->isAllowed( 'phalanxemailblock' ) ) {
+		if ( ( $row->p_type & Phalanx::TYPE_EMAIL ) && !$this->getUser()->isAllowed( 'phalanxemailblock' ) ) {
 			return '';
 		}
 
-		if (isset($row->p_author_id)) {
-			$author = User::newFromId( $row->p_author_id );
+		// SUS-270: Get author from correct field
```

Comment:
This comment probably won't make much sense to someone reading the code (who won't know what the wrong field was). :)